cafebabel/templates/articles/translations/create.html

```diff
@@ -5,16 +5,19 @@
   {{ extra_js(['js/articles-tags.js', 'js/textarea-change.js']) }}
 {% endblock %}
 
+{% block body_class %}create-article-page{% endblock %}
 {% block body %}
-<h1>Creating a translation in {{ language }} for article “{{ article.title }}”</h1>
 
 <form method=post action=.>
   <input type=hidden name=language value="{{ language }}">
   <input type=hidden name=original value="{{ article.id }}">
-  <div class=text>
+  {% if article.has_image %}
+    <img src="{{ article.image_url }}" alt="Image for {{ article.title }}">
+  {% endif %}
+  <h1 class="edit active">
     <label for=title>Title</label>
-    <input id=title name=title required>
-  </div>
+    <input id=title name=title required value="Translation in {{ language }} for article “{{ article.title }}”">
```

Comment:
Isn't it a placeholder? I mean do we want the translation to be saved with such a title?

Reply:
it is a placeholder. And we don't want it to be saved like that. Do you think it's a bad idea to leave it like that?

Reply:
I think using `placeholder=` is more appropriated then?

Reply:
I thought we prohibited `placeholders` because of accessibility, didn't we?

Reply:
It's not the same to use it as a label and to use it as a… placeholder :)

Reply:
ok ! go for the placeholder then !